{{cookiecutter.project_slug}}/docs/installation.rst

```diff
@@ -24,20 +24,21 @@ From sources
 
 The sources for {{ cookiecutter.project_name }} can be downloaded from the `Github repo <{{ cookiecutter.__gh_slug}}>`_.
```

Comment:
We might want an Anaconda version of this file.

Reply:
True. What we can do is stage it via commented out rst code here:

```
..

    Some direction Anaconda directions:

    .. code:: shell

        $ ...
```